modules/developing/pages/clients/flink.adoc

```diff
@@ -0,0 +1,476 @@
+    "error": "{\"type\":\"org.apache.flink.kubernetes.operator.exception.DeploymentFailedException\",\"message\":\"back-off 5m0s restarting failed container=flink-main-container pod=j-flink-tenant-zicjos-84c46c749-pmbfw_fl-flink-tenant(bcaf73fd-0db7-4a5d-897b-63acad0822c3)\",\"additionalMetadata\":{\"reason\":\"CrashLoopBackOff\"},\"throwableList\":[]}",
+    "lifecycleState": "DEPLOYED"
+  }
+}
+----
+--
+====
+
+== View a Flink job's logs
+
+To tail a Flink job's logs:
+[tabs]
+====
+Curl::
++
+--
+[source,bash]
+----
+curl -H "Content-Type: application/json" -H "Authorization: Bearer $token" "https://pulsar-aws-useast2.api.dev.streaming.datastax.com/flink/logs/$tenant/$jobName/jobmanager?tail=true"
+----
+--
+
+Result::
++
+--
+[source,bash]
+----
+no default tenant admin role is defined
+----
+--
+====
+
+To tail a Flink function worker's logs:
+[tabs]
+====
+Curl::
++
+--
+[source,bash]
+----
+curl -H "Content-Type: application/json" -H "Authorization: Bearer $token" "https://pulsar-aws-useast2.api.dev.streaming.datastax.com/flink/logs/$tenant/$jobName/jobmanager?tail=true"
+----
+--
+
+Result::
++
+--
+[source,bash]
+----
+no default tenant admin role is defined
+----
+--
+====
+
+== Delete a Flink job
+
+To delete the job we queried above (`j-flink-tenant-zicjos`), issue a DELETE request to `https://pulsar-aws-useast2.api.dev.streaming.datastax.com/flink/api/jobs/$tenant/j-flink-tenant-zicjos`:
+[tabs]
+====
+Curl::
++
+--
+[source,bash]
+----
+curl -v -H "Content-Type: application/json" -H "Authorization: Bearer $token" -X DELETE $endpoint/$jobName
+----
+--
+
+Result::
++
+--
+[source,bash]
+----
+> DELETE /flink/api/jobs/flink-tenant/j-flink-tenant-zicjos HTTP/1.1
+> Host: pulsar-aws-useast2.api.dev.streaming.datastax.com
+> User-Agent: curl/7.79.1
+> Accept: */*
+> Content-Type: application/json
+> Authorization: Bearer AstraCS:xxxx
+>
+* Mark bundle as not supporting multiuse
+< HTTP/1.1 200 OK
+----
+--
+====
+
+== Alternate method: Test a Flink query with a consumer
+
+You can create a consumer to check the status of the Flink job. The consumer will read from the "flink-tenant/flink/$jobName" topic, which is created when you create the Flink job.
+
+. Create a consumer to read from the "flink-tenant/flink/$jobName" topic:
+[tabs]
+====
+Console::
++
+--
+[source,bash]
+----
+cat > create-job-persons.json << EndOfMessage
+{
+  "parallelism": 1,
+  "createTableSpecs": [
+{
+ "tableName": "input", 
+ "format": "json",
+ "topic": "$inputTopic",
+ "columns": ["name string"]  
+}
+],
+  "selectQuery": "select * from flink.input as input"
+}
+EndOfMessage
+cat create-job-persons.json | jq
+endpoint="https://pulsar-aws-useast2.api.dev.streaming.datastax.com/flink/api/previews/$tenant"
+response=$(curl -H "Content-Type: application/json" -H "Authorization: Bearer $token" -X POST  --data @create-job-persons.json $endpoint -v -o -)
+outputTopic=$(echo "$response" | jq -c '.specs.outputTable' | tr -d '`' | tr -d '"' | awk -F '\.' '{print $2}')
```

Comment:
Should we change "outputTopic" to "jobTopic". This will avoid confusion with the previous example.

Also, it is better to separate the job creation and execution scripts from the consumer creation script. It will be also helpful to show the result of this jobTopic